### PR TITLE
Fixes #23 adds license file and updates readme file

### LINKED ISSUE #23: Publish element to webcomponents.org

### PR DESCRIPTION
**Pull request by anushagadepalli**

### CODE REVIEW

**Comment by anushagadepalli** [comment]
@cherukumilli 
I updated the license file. Can you please review my code and merge it in? 
So that I can publish the element and add the webcomponents.org badge.

File: .travis.yml [hunk 1]
```diff
@@ -1,21 +1,18 @@
 language: node_js
-node_js: 
-  - "4"
+node_js: stable
```
Comment (cherukumilli) [inline .travis.yml]: @anushagadepalli 
why do we need these changes?
we should be using node 6 instead of stable

File: .travis.yml [hunk 2]
```diff
@@ -1,21 +1,18 @@
...
 dist: trusty
 sudo: required
 addons:
   firefox: latest
-  sauce_connect: true
```
Comment (cherukumilli) [inline .travis.yml]: @anushagadepalli 
why?

File: .travis.yml [hunk 3]
```diff
@@ -1,21 +1,18 @@
...
   apt:
     sources:
     - google-chrome
     packages:
     - google-chrome-stable
 before_script:
-- npm install -g bower polylint web-component-tester eslint eslint-plugin-html
+- npm install -g bower polylint web-component-tester
```
Comment (cherukumilli) [inline .travis.yml]: why?

File: .travis.yml [hunk 4]
```diff
@@ -1,21 +1,18 @@
...
 - bower install
 - polylint
-- eslint *.html test/*.html
```
Comment (cherukumilli) [inline .travis.yml]: why?

**Comment by anushagadepalli** [comment]
@cherukumilli 
I updated .travis.yml file, can you please review my code?

**Pull request merged**